src/main/java/org/opensrp/domain/LocationAndStock.java

```diff
@@ -0,0 +1,20 @@
+package org.opensrp.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.opensrp.domain.postgres.Location;
```

Comment:
The class uses `domain.postgres.Stock` and `domain.postgres.Location` domain from `server-core` they have the `json` attribute in them so i dont think it should be from PlanEvaluatorLib (since they cant be reused by the client) .. However this class `PhysicalLocationAndStocks` is from PlanEvaluator.